Assignments/Program.cs

```diff
@@ -1,3 +1,50 @@
-﻿using System;
+﻿// ---------------------------------------------------------------------------------------
+// Spark23 Assignments
+// Copyright (c) Metamation India.
+// ---------------------------------------------------------------------------------------
+// Program.cs
+// T16 - Longest Abcdedarian Word
+// ---------------------------------------------------------------------------------------
 
-Console.WriteLine("Hello, World!");
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Spark;
+public class T16 {
+   public static void Main () {
+      Console.WriteLine ("\x1B[4m" + "Longest Abcdedarian Finder:-" + "\x1B[0m");
+      List<string> words = new () { "apple", "first", "antwz", "efghijlk", "pqrst", "orange" };
+      words = words.OrderBy (a => a.Length).ToList ();
+      PrintResult (words);
+      Console.WriteLine ("\nNow, let's try words 3 different words:- ");
+      words.Clear ();
+      for (int i = 0; i < 3; i++) {
+         Console.Write ($"Enter words {i + 1}: ");
+         words.Add (Console.ReadLine ());
+      }
+      PrintResult (words);
+   }
+
+   /// <summary>Returns the Longest Abcdedarian Word from the given list of words</summary>
+   public static string LongestAbcdedarianWordOf (List<string> input) {
+      input.Sort ();
+      foreach (string word in input)
+         if (word == string.Concat (word.OrderBy (a => a))) return word;
+      return "";
+   }
+
+   /// <summary>Prints the result whether the words is isogram or not</summary>

```

Comment:
```suggestion
   /// <summary>Prints the result whether the words is isogram or not</summary>

```
Again copy paste error. This is not isogram method